src/data/move-attrs/present-power-attr.ts

```diff
@@ -19,16 +19,16 @@ export class PresentPowerAttr extends VariablePowerAttr {
      * If this move is multi-hit, and this attribute is applied to any hit
      * other than the first, this move cannot result in a heal.
      */
-    const firstHit = user.turnData.hitCount === user.turnData.hitsLeft;
+    const isFirstHit = user.turnData.hitCount === user.turnData.hitsLeft;
 
-    const powerSeed = randSeedInt(firstHit ? 100 : 80);
+    const powerSeed = user.randSeedInt(isFirstHit ? 100 : 80);
     if (powerSeed <= 40) {
       power.value = 40;
     } else if (40 < powerSeed && powerSeed <= 70) {
       power.value = 80;
     } else if (70 < powerSeed && powerSeed <= 80) {
```

Comment:
```suggestion
    if (powerSeed < 40) {
      power.value = 40;
    } else if (powerSeed < 70) {
      power.value = 80;
    } else if (powerSeed < 80) {
```
(`powerSeed` ranges from 0 to 99.)